docs/set-up.asciidoc

```diff
@@ -6,34 +6,90 @@ you must first {apm-server-ref-v}/configuration-rum.html[enable the RUM endpoint
 
 Once the APM Server endpoint has been configured, you can:
 
-* <<install-the-agent>>
+* <<usage>>
 * <<configuring-cors>>
 
-[[install-the-agent]]
-=== Install the Agent
+[[usage]]
+=== Usage
```

Comment:
Let's try to keep these as actions. Here's a screenshot:
<img width="500" alt="Screen Shot 2020-04-08 at 8 22 49 AM" src="https://user-images.githubusercontent.com/5618806/78801988-27409d00-7972-11ea-9e82-3b35b956aed7.png">

Instead of "usage" replacing "install the agent", would any of these make sense:
* Import the Agent?
* Load the Agent?
* Initialize the Agent?
* Include the Agent?

Reply:
Initialise/Import sounds good.